amundsen_application/static/js/components/TableDetail/EditableSection/index.tsx

```diff
@@ -4,6 +4,7 @@ import './styles.scss';
 
 export interface EditableSectionProps {
   title: string;
+  readOnly: boolean;
```

Comment:
Lets make this optional with `readOnly?: boolean`. It will allow the default version of this component to be editable and we only have to throw `readOnly={true}` on the new non-editable cases.

Reply:
ah ok thats how you do that :) great, fixing.

Reply:
do I need to set equal to false by default in the constructor? or will that be the default value?

Reply:
i believe the answer is that it is false by default. Everything appears to work ok as is, so resolving.